### PR TITLE
Add GUI Editor to Configuration view

### PR DESCRIPTION
**Pull request by ashmaroli**
- [x] add GUI to `Configuration` view
- [x] update documentation
- [x] update spec files

### CODE REVIEW

File: src/containers/views/Configuration.js [hunk 1]
```diff
@@ -61,16 +72,33 @@ export class Configuration extends Component {
         {errors && errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
           <h1>Configuration</h1>
-          <div className="page-buttons">
+          <div className="page-buttons multiple">
+            <Button // TODO: Hide toggle for non-YAML config files (e.g. '_config.toml')
+              onClick={this.toggleView.bind(this)}
```
Comment (mertkahyaoglu) [inline src/containers/views/Configuration.js]: Since we started binding `this` in the `constructor` with `handleClickSave`, let's keep doing it for other ones as well.

File: src/containers/views/Configuration.js [hunk 2]
```diff
@@ -61,16 +72,33 @@ export class Configuration extends Component {
...
+              active={editorChanged || fieldChanged}
+              triggered={updated}
+              block />
           </div>
         </div>
+          {
+            this.state.guiView && content &&
+              <div className="content-body">
+                <div className="warning">
+                  CAUTION! Any existing comments and formatting will be lost when editing via this view.
```
Comment (mertkahyaoglu) [inline src/containers/views/Configuration.js]: Shall we make this lang constant? It appears more than one places now.

Comment (ashmaroli) [inline src/containers/views/Configuration.js]: :+1:  but should it be covered in a separate PR to discuss the translated versions?

Comment (mertkahyaoglu) [inline src/containers/views/Configuration.js]: 👍

**Comment by mertkahyaoglu** [comment]
Great! I think GUI for `Configuration` is the most useful one ❤️
